shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -138,6 +138,11 @@ export default {
         hash.allPSPs = await this.$store.dispatch('management/findAll', { type: MANAGEMENT.POD_SECURITY_POLICY_TEMPLATE });
       }
 
+      // TODO: Check if getters and resources are right
```

Comment:
The changes in this file look like code from https://github.com/rancher/dashboard/pull/7787